Assets/Scripts/Tools/AreaSelection.cs

```diff
@@ -1,11 +1,12 @@
-using System;
 using System.Collections;
 using System.Collections.Generic;
 using System.IO;
 using System.Runtime.InteropServices;
+// ReSharper disable once RedundantUsingDirective
```

Comment:
The System.text using statement is only used inside an `#if`; so Rider does not 'see' it. Removing it will cause a build failure so I added this directive for Rider to ignore that it _thinks_ the using statement is unused